documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -1,92 +1,583 @@
 ---
-title: "Install the operator"
+title: "Installation and upgrade"
 date: 2019-02-23T16:47:21-05:00
-weight: 1
-
+weight: 2
+description: "How to install, upgrade, and uninstall the operator."
 ---
 
-The operator uses Helm to create the necessary resources and
-then deploy the operator in a Kubernetes cluster. This document describes how to install, upgrade,
-and uninstall the operator.
-
-#### Contents
-
- - [Operator image](#operator-image)
- - [Install the operator Helm chart](#install-the-operator-helm-chart)
- - [Alternatively, install the operator Helm chart from the GitHub chart repository](#alternatively-install-the-operator-helm-chart-from-the-github-chart-repository)
- - [Upgrade the operator](#upgrade-the-operator)
- - [Uninstall the operator](#uninstall-the-operator)
+### Contents
 
- #### Operator image
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Install the operator](#install-the-operator)
+- [Set up domain namespaces](#set-up-domain-namespaces)
+- [Update a running operator](#update-a-running-operator)
+- [Upgrade the operator](#upgrade-the-operator)
+  - [Upgrading a 2.6 operator to a 3.x operator](#upgrading-a-26-operator-to-a-3x-operator)
+  - [Upgrading a 3.x operator](#upgrading-a-3x-operator)
+- [Uninstall the operator](#uninstall-the-operator)
+- [Installation sample](#installation-sample)
 
- You can find the operator image in
- [GitHub Container Registry](https://github.com/orgs/oracle/packages/container/package/weblogic-kubernetes-operator).
+### Introduction
 
-#### Install the operator Helm chart
+This installation guide describes how to configure, install (deploy), update, upgrade,
+and uninstall an instance of the WebLogic Kubernetes Operator.
+A single instance is capable of managing multiple domains in multiple namespaces, depending on how it is configured.
+A Kubernetes cluster can host multiple operators, but no more than one per namespace.
+
+### Prerequisites
+
+Before installing an operator, ensure that each of these prerequisite requirements are met:
```

Comment:
Change "are met" to "is met"?

Reply:
Not sure. I think either works. @rosemarymarano ?

Reply:
Is met (I think that was my mistake)

Reply:
done